doc/site/zh-cn/guides/thread-primer/ipv6-addressing.md

```diff
@@ -0,0 +1,231 @@
+
+常见的单播类型在下面详细介绍。
+
+<table>
+  <tbody>
+    <tr>
+      <th colspan=2><h3>Link-Local Address (LLA)</h3></th>
+    </tr>
+    <tr>
+      <td colspan=2 style="background-color:rgb(238, 241, 242)">一个 EID，用于标识单个无线电传输可到达的 Thread 接口。</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>示例</b></td><td><code>fe80::54db:881c:3845:57f4</code></td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>IID</b></td><td>基于 802.15.4 Extended Address（扩展地址）</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>域</b></td><td>Link-Local</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>详情</b></td><td><ul><li>用于发现邻居、配置链接和交换路由信息</li><li>非可路由地址</li><li>总是带 <code>fe80::/16</code> Prefix</li></ul></td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <th colspan=2><h3>Mesh-Local EID (ML-EID)</h3></th>
+    </tr>
+    <tr>
+      <td colspan=2 style="background-color:rgb(238, 241, 242)">一个 EID，用于标识 Thread 接口。用于访问同一 Thread 分区内的 Thread 接口。也称为 ULA（Unique Local Address）。</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>示例</b></td><td><code>fde5:8dba:82e1:1:416:993c:8399:35ab</code></td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>IID</b></td><td>随机的，在 commissioning 后选定</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>域</b></td><td>Mesh-Local</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>详情</b></td><td><ul><li>不会随拓扑变化而发生变更</li><li>应由应用程序使用</li><li>总是带 <code>fd00::/8</code> Prefix</li></ul></td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <th colspan=2><h3>Routing Locator (RLOC)</h3></th>
+    </tr>
+    <tr>
+      <td colspan=2 style="background-color:rgb(238, 241, 242)">根据 Thread 接口在网络拓扑中的位置来对其进行标识。</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>示例</b></td><td><code>fde5:8dba:82e1:1::ff:fe00:1001</code></td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>IID</b></td><td><code>0000:00ff:fe00:<var>RLOC16</var></code></td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>域</b></td><td>Mesh-Local</td>
+    </tr>
+    <tr>
+      <td width="25%" style="background-color:rgb(238, 241, 242)"><b>详情</b></td><td><ul><li>在设备连接到网络后生成</li><li>用于在 Thread 网络中传递 IPv6 数据报</li><li>随拓扑变化而发生变更</li><li>通常不会由应用程序使用</li></ul></td>
+    </tr>
+  </tbody>
+</table>
+
+<table>
+  <tbody>
+    <tr>
+      <th colspan=2><h3>Anycast Locator (ALOC)</h3></th>
+    </tr>
+    <tr>
+      <td colspan=2 style="background-color:rgb(238, 241, 242)">当目标的 RLOC 未知时，通过 RLOC 查找标识 Thread 接口。</td>
```

Comment:
```suggestion
      <td colspan=2 style="background-color:rgb(238, 241, 242)">如果不知道目标的 RLOC, 使用 ALOC 地址, 通过查找相应的 RLOC, 也可以标识 Thread 接口。</td>
```

Reply:
I think the original text is a bit unclear.

By consulting the definition of the Thread specification, I think the description in the specification will be more appropriate:

```
A Thread Anycast Locator (ALOC) is an IPv6 address that identifies the location of one or more Thread interfaces within a Thread Network Partition. ALOCs are used where the specific RLOC of a destination is not known at the originator.
```

Translated as:

```
用于标识 Thread 网络分区中一个或多个 Thread 接口的位置。如果始发者不知道目的地的 RLOC，则使用 ALOC 进行查找。
```

What do you think?

Reply:
`RLOC look up` happens in the sender itself to find/fill the mesh layer dest via the Thread Network Data when forwarding. maybe could explain `RLOC look up` a bit to avoid some confusion when reading?

Reply:
I don't think this is necessary, because RLOC has already been described above.

Reply:
+1
Fine if you think RLOC loop up itself is clear enough. we can simply keep the translation same with the english source.